app/presenters/sort_presenter.rb

```diff
@@ -0,0 +1,8 @@
+class SortPresenter
+  attr_reader :q, :sort_key, :sort_direction
+
+  def initialize(sort_key:, sort_direction:)
+    @sort_key = sort_key
+    @sort_direction = sort_direction.to_sym
+  end
+end
```

Comment:
Appreciate that this isn't particularly useful at present. Will keep for now but will definitely be refactoring.